modules/govuk_jenkins/manifests/jobs/data_sync_complete_staging.pp

```diff
@@ -16,9 +16,32 @@
   $auth_token = undef,
   $signon_domains_to_migrate = [],
 ) {
+  if $::aws_migration {
+    $aws = true
+    $carrenza = false
```

Comment:
Do we need this variable?

Reply:
Ah, no, I was looking at making parts conditional to only run on Carrenza, but then decided against it. I've now removed the variable.